src/components/Header.js

```diff
@@ -1,12 +1,14 @@
 import { NavLink } from 'react-router-dom';
+import userIcon from '../assets/userIcon.svg';
 
 const Header = () => (
-  <header>
-    <h1>Bookstore CMS</h1>
-    <ul>
-      <li><NavLink to="/">BOOKS</NavLink></li>
-      <li><NavLink to="/categories">CATEGORIES</NavLink></li>
+  <header className="header-container">
+    <h1 className="header-title">Bookstore CMS</h1>
+    <ul className="header-nav">
```

Comment:
- [ ] Dear @JuanDiegoGuerra, For the font family, I notice you have not applied any fonts, as you can see from the `zeplin` design, the main font for the heading is `Montserrat` please apply the font family. 

### Your design 
![Screen Shot 2023-08-25 at 6 05 22 PM](https://github.com/JuanDiegoGuerra/Bookstore/assets/69077061/05d9309b-11a4-49cf-82ff-d4fdd715898f)
### Original design 
![Screen Shot 2023-08-25 at 6 06 17 PM](https://github.com/JuanDiegoGuerra/Bookstore/assets/69077061/9776dabd-fe57-489f-9197-7dd6305cb631)

- [ ] Also, the original design is a bit smaller in font sizing, Please consider making some adjustments to the sizing also.